layers/+lang/elixir/packages.el

```diff
@@ -124,8 +124,8 @@
                    (cons 'elixir-enable-compilation-checking nil))
       (add-to-list 'safe-local-variable-values
                    (cons 'elixir-enable-compilation-checking t))
-      (add-hook 'elixir-mode-hook
-                'spacemacs//elixir-enable-compilation-checking t))))
+      (spacemacs/add-to-local-vars-hook 'elixir-mode-hook
```

Comment:
Judging from the two functions in spacemacs-base/funcs.el you have to use `elixir-mode` here?